featurex/stimuli/video.py

```diff
@@ -12,7 +12,8 @@ class VideoFrameStim(ImageStim):
 
     def __init__(self, video, frame_num, duration=None, filename=None, data=None):
         self.video = video
-        self.data = self.video.frames[frame_num]
+        if data is None:
+            self.data = self.video.get_frame(index=frame_num)
```

Comment:
@tyarkoni  This is problematic because `get_frame` returns a `VideoFrameStim` rather than image data. Should it return the data instead, or should this line have `.data` tacked to the end?